quic/s2n-quic-core/src/recovery/bbr/congestion.rs

```diff
@@ -65,6 +65,8 @@ impl State {
         }
 
         if self.loss_round_counter.round_start() {
+            // TODO: Check for ecn_in_round as in bbr2_adapt_lower_bounds
```

Comment:
Do we need an issue for this or are you going to do a fast follow?

Reply:
I'll make an issue and link it here, at this point I need to research a bit how this works in the Linux TCP impl to see if its worth it

Reply:
#1423